Dockerfile

```diff
@@ -1,7 +1,6 @@
 FROM scratch
-ARG PROJECT_NAME="oasbinder"
-ENV PROJECT_NAME=${PROJECT_NAME}
-COPY ${PROJECT_NAME} /
+COPY oasbinder /
 WORKDIR /
 # hadolint ignore=DL3025
-ENTRYPOINT ${PROJECT_NAME}
+EXPOSE 8080
+ENTRYPOINT ["/oasbinder"]
```

Comment:
It seems it's necessary to change entrypoint to this because:
* the image doesn't have a shell, so the shell form of entrypoint won't work
* env variables aren't substituted in exec form of entrypoint